docs/getting-started/understanding-config-files.md

```diff
@@ -465,31 +465,45 @@ leave a node vulnerable to eclipse attacks.
 
 The main method for configuring peer sharing involves setting the following option:
 
-- `PeerSharing` (default value: `PeerSharingDisabled`) - this option can take two possible values:
-    * `PeerSharingDisabled`: peer sharing is disabled, which means the node won't request peer
+- `PeerSharing` (a boolean value, default `False`)
+    * `False`: peer sharing is disabled, which means the node won't request peer
       information from any other node, and will not respond to such requests from others
       (the mini-protocol won't even start);
-    * `PeerSharingEnabled`: peer Sharing is enabled and the node will notify other nodes
+    * `True`: peer sharing is enabled and the node will notify other nodes
```

Comment:
The rest of the sentence is incorrect.
`True`: peer sharing is enabled and the node may issue as well as respond to request for peers.